src/app/clusters/basic/basic.cpp

```diff
@@ -32,6 +33,21 @@ void emberAfBasicClusterServerInitCallback(chip::EndpointId endpoint)
 {
     EmberAfStatus status;
 
+    char nodeLabel[DeviceLayer::ConfigurationManager::kMaxNodeLabelLength + 1];
+    if (ConfigurationMgr().GetNodeLabel(nodeLabel, sizeof(nodeLabel)) == CHIP_NO_ERROR)
+    {
+        status = Attributes::NodeLabel::Set(endpoint, chip::CharSpan(nodeLabel, strlen(nodeLabel)));
+        VerifyOrReturn(EMBER_ZCL_STATUS_SUCCESS == status, ChipLogError(Zcl, "Error setting Node Label: 0x%02x", status));
```

Comment:
Is returning early the best strategy we can get here?

I would suggest to just log and keep going trying to set the rest. I am unsure if a failure to set say 'node label' should make it so that  no other attempts to setting things are possible.

Reply:
Yes, you are right, I will change it. Old code is using early returning, and I Ignored it.